src/python_routing_v02/troute/routing/fast_reach/mc_reach.pyx

```diff
@@ -349,103 +353,117 @@ cpdef object compute_network(
     # minimal validation,
     # Jump straight to nogil.
 
-    with nogil:
```

Comment:
We have to support the nogil

Reply:
I tried adding cdef comments for all available variables that are erroring. However, I am still receiving plenty of python conversion, Coercion, and indexing errors.